test/end-to-end/helpers/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const Chance = require('chance');
+
+const chance = new Chance();
+
+const helpers = {};
+
+helpers.getAuthClientParams = function() {
+  return {
+    clientId: 'e922556a-7d4f-4168-88cd-059276044798',
+    clientSecret: '79c07401-d3b2-48c0-8407-dc19c4ece7ff',
+    redirectUri: 'http://localhost:4040/callback',
+    development: true,
+  };
+};
+
+const getCodeFromUrl = function(uri) {
```

Comment:
Nit: Naming mismatch. Either name the function `getCodeFromUri` or name the parameter variable `url`.